doc/managing-wallets.md

```diff
@@ -108,7 +108,11 @@ Wallets created before version 0.13 are not HD and must be backed up every 100 k
 
 ### 1.6 Restoring the Wallet From a Backup
 
-To restore a wallet, the `restorewallet` RPC must be used.
+To restore a wallet, the `restorewallet` RPC or the `Restore Wallet` GUI menu item must be used.
+
+In the GUI, a backup can be restored via `File` -> `Restore Wallet...`.
```

Comment:
Following the format of the "1.4 Backing Up the Wallet" section, I believe this is better to go bellow the RPC command https://github.com/bitcoin/bitcoin/blob/fbe5e1220a35c35a668ec965c66508a7839aa7dd/doc/managing-wallets.md?plain=1#L114